test/test_automl/test_automl.py

```diff
@@ -151,7 +156,7 @@ def test_delete_non_candidate_models(self):
         X, Y, _, _ = putil.get_dataset('iris')
         automl = autosklearn.automl.AutoML(
             backend_api,
-            time_left_for_this_task=30,
+            time_left_for_this_task=40,
```

Comment:
What's the reason to update this?

Reply:
No Reason, and I have reverted this. I forgot to set back while debugging